n2y/notion.py

```diff
@@ -201,11 +201,13 @@ def wrap_notion_file(self, notion_data):
     def wrap_notion_emoji(self, notion_data):
         return self.instantiate_class("emoji", None, self, notion_data)
 
-    def wrap_notion_rich_text_array(self, notion_data, block=None):
-        return self.instantiate_class("rich_text_array", None, self, notion_data, block)
+    def wrap_notion_rich_text_array(self, notion_data, block=None, table=None):
```

Comment:
It looks like `table` is a boolean everywhere else. Should we set a default boolean value rather than None here? Same thing below.